app/components/elements/spinner_component.rb

```diff
@@ -4,7 +4,7 @@ module Elements
   # It spins.
   class SpinnerComponent < ApplicationComponent
     def initialize(message: 'Loading...', image_path: nil, variant: nil, hide_message: false, classes: [], # rubocop:disable Metrics/ParameterLists
-                   height: nil, width: nil, message_classes: [])
+                   height: nil, width: nil, message_classes: [], speed: 0.75)
```

Comment:
Worth a comment pointing at https://getbootstrap.com/docs/5.3/components/spinners/#sass-variables to show `0.75` is the default value in Bootstrap?

Reply:
Good idea. Adding.